go.mod

```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/Masterminds/sprig/v3 v3.2.0
-	github.com/blang/semver/v4 v4.0.0
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
Any idea why we are downgrading here?

Reply:
I think it's because we don't import `v4` explicitly anywhere in the code. I'll update it.